bigquery/samples/tests/test_client_query_destination_table_legacy.py

```diff
@@ -13,15 +13,13 @@
 # limitations under the License.
 
 
-from .. import get_routine
+from .. import client_query_destination_table_legacy
 
 
-def test_get_routine(capsys, client, routine_id):
+def test_client_query_destination_table_legacy(capsys, client, table_id):
```

Comment:
Use `random_table_id` since we want to create a new table in the actual sample.

The `table_id` creates a table before the tests run. https://github.com/googleapis/google-cloud-python/blob/93600571c88d31c94d5080af35541d27b0021443/bigquery/samples/tests/conftest.py#L76